backend/src/main/java/com/senior/project/backend/Activity/TaskRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.senior.project.backend.Activity;
+import com.senior.project.backend.domain.Task;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface TaskRepository extends JpaRepository<Task, Long> {
+    // additional query methods if needed
+    @Query("SELECT t FROM Task t")
+    List<Task> findAll();
```

Comment:
why do you need this? you should be able to just use the default findall() method from jpaRepository

Reply:
artifact from copying the code from milestone. i think i'll leave it in case we want to make the query more complicated later